homeassistant/components/homeassistant/exposed_entities.py

```diff
@@ -442,25 +444,41 @@ def ws_list_exposed_entities(
     result: dict[str, Any] = {}
 
     exposed_entities = hass.data[DATA_EXPOSED_ENTITIES]
-    required_assistant = msg.get("assistant")
     entity_registry = er.async_get(hass)
     for entity_id in chain(exposed_entities.entities, entity_registry.entities):
-        entity_settings = async_get_entity_settings(hass, entity_id)
```

Comment:
should we revert this in a separate PR (and also revert the tests from that PR)

Reply:
Or just change the title of this PR to reflect that it is replacing and not adding

Reply:
Yes, I agree we should revert instead.
Also, it turns out there might be a bug in `homeassistant/expose_entity/list`, it seems it was never meant to send all entities.